build/grunt.js

```diff
@@ -397,11 +448,18 @@ module.exports = function(grunt) {
     'copy:swf',
     'copy:ie8',
     'vjslanguages'
-  ]);
+  ];
+
+  grunt.registerTask('build', buildDependents);
+
+  grunt.registerTask(
+    'build:dist',
+    buildDependents.map(task => task === 'browserify:build' ? 'browserify:dist' : task)
```

Comment:
so, basically replace `browserify:build` with `browserify:dist` but keep everything the same?